dataset/bin.cpp

```diff
@@ -197,6 +198,14 @@ template <class T, class Meta> auto extract_unbinned(T &array, Meta meta) {
   return extracted;
 }
 
+/// Combine meta data from buffer and input data array and create final output
+/// data array with binned data.
+/// - Meta data that does not depend on the buffer dim is lifted to the output
+///   array.
+/// - Any meta data depending on rebinned dimensions is dropped since it becoms
```

Comment:
typo becoms -> becomes

Reply:
I'll fix this elsewhere, so we don't have to rerun CI.